pyproject.toml

```diff
@@ -20,12 +20,12 @@ classifiers = [
     "Programming Language :: Python",
     "Programming Language :: Python :: 3",
     "Programming Language :: Python :: 3 :: Only",
-    "Programming Language :: Python :: 3.7",
     "Programming Language :: Python :: 3.8",
     "Programming Language :: Python :: 3.9",
     "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
 ]
-requires-python = ">=3.7"
+requires-python = ">=3.8"
```

Comment:
Flagging for public awareness! :)

Reply:
Thanks for flagging, @vidartf.

@blink1073 you mentioned that SciPy ecosystem has a policy where they support the last few versions, even if the oldest version hasn't reach end-of-life. Do you have a link to something stating this policy? It would be nice to point to it, in case folks have concerns about this bump. 

I found this support table from Numpy: https://numpy.org/neps/nep-0029-deprecation_policy.html

This shows that Numpy will drop support for 3.8+ in December.

Reply:
Here's the link from conda-forge: https://conda-forge.org/docs/user/announcements.html#dropping-python-3-7

Reply:
Note this drop is only for the 2.x branch, we will still keep support in 1.x

Reply:
Thank you!